mypy.ini

```diff
@@ -0,0 +1,16 @@
+[mypy]
+packages = elementary
+exclude = ^elementary/monitor/dbt_project/
+allow_redefinition = True
```

Comment:
This means you can do something like:
```
res = [... some list comprehension ... ].    # Type is List[str]
res = json.dumps(res)                                # Type is str - mypy by default alerts when that happens
```

I thought in some cases it is legitimate and online that other people also allowed this in their projects. But we can also be more strict

Reply:
I think that we shouldn't allow this.
Just confusing in my opinion and better to call the new variable `res_dump` for instance.
It highlights the change and is easier to understand.

Reply:
Hmm ok I'll try to remove it and see how many cases it produces